classes/components/forms/context/PKPContactForm.inc.php

```diff
@@ -66,7 +66,7 @@ public function __construct($action, $locales, $context) {
 			]))
 			->addField(new FieldTextarea('mailingAddress', [
 				'label' => __('common.mailingAddress'),
-				'isRequired' => true,
+				'isRequired' => false,
```

Comment:
This PR is for pkp/pkp-lib#7651, but this is part of https://github.com/pkp/pkp-lib/issues/2169; please keep them separate.